cardano-api/src/Cardano/Api/Tx.hs

```diff
@@ -433,6 +432,7 @@ getTxBody (ShelleyTx era tx) =
                     (Map.elems msigWits)
                     TxBodyNoScriptData
                     (strictMaybeToMaybe txAuxiliaryData)
+                    ScriptValid
```

Comment:
Hmm we should probably implement a GADT similar to `TxBodyScriptData era`.

Reply:
Done!